src/App/App.js

```diff
@@ -610,7 +633,7 @@ function FullApp() {
             <Trans>WalletConnect</Trans>
           </div>
         </button>
-        {active && approvalsButton === true && (
+        {active === true && hasTokens === true && (
```

Comment:
I believe `active` and `hasTokens` are always booleans, so the simplified `active && hasTokens` should be sufficient. But if there is an edge case please disregard.